runtime/src/commonJvmAndroid/kotlin/pbandk/internal/binary/InputStreamWireReader.kt

```diff
@@ -0,0 +1,385 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Since you  have the license at the top, does that mean files are copied as is?  So no need to review?

Reply:
Yeah. The files with the Google license are basically a direct Java-to-Kotlin conversion of code from the protobuf-java library. I only made minor Kotlin syntax tweaks and tried not to change any of the logic.